roxie/ccd/ccdmain.cpp

```diff
@@ -717,6 +717,20 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
         if (udpSnifferEnabled && !roxieMulticastEnabled)
             DBGLOG("WARNING: ignoring udpSnifferEnabled setting as multicast not enabled");
 
+        int ttlTmp = topology->getPropInt("@multicastTTL", 1);
```

Comment:
If ttlTmp is there to be able to catch the incorrect -ve values, then you can leave it as is.